### ISSUE TITLE
html output problems: id

### ISSUE THREAD

**Issue by peter1000** [MichaelHatherly/Lexicon.jl#65]
eventhough html output might be replaced: might be still good to open an issue.

generated id do not pass validation: 
e.g. `<div id='save(file::AbstractString, modulename::Module)' class='entry-name category-method'>`
http://validator.w3.org/#validate_by_input+with_options

http://www.w3schools.com/tags/att_global_id.asp
```
id 	Specifies a unique id for the element. Naming rules:

    Must contain at least one character
    Must not contain any space characters
    In HTML, all values are case-insensitive
```

**Comment by MichaelHatherly** [comment]
Would we just need to escape each id I assume? I think there may be an escape function in the markdown.jl or base/markdown.jl code bases that we could use.

**Comment by peter1000** [comment]
This one I could fix but I was unsure if it is worthy as you said you will change to something and it does work as it is - just if you want to stick with it it should got fixed.

Just let me know.

**Comment by MichaelHatherly** [comment]
If you'd like to, then go for it. Can definitely wait until the larger PR is merged though

**Comment by peter1000** [comment]
Ok I do that later.

**Comment by peter1000** [comment]
Just for info: I will do this next.

**Comment by MichaelHatherly** [comment]
Closed by #77.
